tests/unit/requirements.txt

```diff
@@ -0,0 +1,2 @@
+junit_xml
+semver
```

Comment:
@fredericlepied we're currently re-writing ocp_compatibility.py with @pierreblanc to get rid of semver: https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/247/files

Reply:
would it be fine if we first merge another PR and then re-test yours?